### PR TITLE
Import MaskedArray from numpy

### PR DESCRIPTION
**Pull request by mrocklin**
The sklearn import has broken

### CODE REVIEW

**Comment by mrocklin** [comment]
I'm on 0.18 (.0?) .  This seems to be latest on conda-forge?

mrocklin@carbon:~$ anaconda search scikit-learn | grep conda-forge
Using Anaconda Cloud api site https://api.anaconda.org
     conda-forge/scikit-learn  |     0.18 | conda           | linux-64,
osx-64

Perhaps try-except between sklearn and numpy?

**Comment by jcrist** [comment]
The fix in scikit-learn is needed to be able to pickle the masked arrays, but that's it, so a try/catch block would probably be fine. You're correct, 0.18.0 is the latest on conda-forge, but not the latest release. Not sure why conda-forge isn't up to date yet.

**Comment by jcrist** [comment]
This was fixed in #37, it now works with scikit-learn 0.18.0+. Closing.
